docs/manual/src/docs/asciidoc/_includes/about/authentication/password-storage.adoc

```diff
@@ -147,16 +146,16 @@ All of the original passwords are "password".
 ----
 ====
 
-<1> The first password would have a `PasswordEncoder` id of `bcrypt` and encodedPassword of `$2a$10$dXJ3SW6G7P50lGmMkkmwe.20cQQubK3.HZWzG3YB1tlRy.fqvM/BG`.
-When matching it would delegate to `BCryptPasswordEncoder`
-<2> The second password would have a `PasswordEncoder` id of `noop` and encodedPassword of `password`.
-When matching it would delegate to `NoOpPasswordEncoder`
-<3> The third password would have a `PasswordEncoder` id of `pbkdf2` and encodedPassword of `5d923b44a6d129f3ddf3e3c8d29412723dcbde72445e8ef6bf3b508fbf17fa4ed4d6b99ca763d8dc`.
-When matching it would delegate to `Pbkdf2PasswordEncoder`
-<4> The fourth password would have a `PasswordEncoder` id of `scrypt` and encodedPassword of `$e0801$8bWJaSu2IKSn9Z9kM+TPXfOc/9bdYSrN1oD9qfVThWEwdRTnO7re7Ei+fUZRJ68k9lTyuTeUp4of4g24hHnazw==$OAOec05+bXxvuu/1qZ6NUR+xQYvYv7BeL1QxwRpY5Pc=`
-When matching it would delegate to `SCryptPasswordEncoder`
-<5> The final password would have a `PasswordEncoder` id of `sha256` and encodedPassword of `97cde38028ad898ebc02e690819fa220e88c62e0699403e94fff291cfffaf8410849f27605abcbc0`.
-When matching it would delegate to `StandardPasswordEncoder`
+<1> The first password has a `PasswordEncoder` id of `bcrypt` and an encoded password value of `$2a$10$dXJ3SW6G7P50lGmMkkmwe.20cQQubK3.HZWzG3YB1tlRy.fqvM/BG`.
```

Comment:
I have no way to address this one, since I don't know where the value would go, and I'm not in the business of guessing.

I was editing the words, not the code.

Reply:
For these [changes](https://onevmw-my.sharepoint.com/:w:/g/personal/rwinch_vmware_com/Ed-v0AUZNalPi_0PTR7k2zcBzfGH46YvqIE_biTc7zaaHw?e=dMON3B) change encodedPassword to be `encodedPassword` instead of encoded password because it is referring to [`{id}encodedPassword`](https://github.com/spring-projects/spring-security/blame/ca7f27a530c973de394441af9299c19d7b1cd289/docs/manual/src/docs/asciidoc/_includes/about/authentication/password-storage.adoc#L127)

Reply:
OK. I put them back to encodedPassword and marked them as code.